docs: add current state

fuzzylogic2000 opened the pull request.

phillimorland commented on docs/currentState.md:
```diff
@@ -0,0 +1,110 @@
+- name
+- description as plain text
+- categories if they exist (but whenever the category is clicked we get `VirtualizedLists should never be nested inside plain ScrollViews with the same orientation - use another VirtualizedList-backed container instead.`)
+- labels if they exist
```
labels are not yet styled correctly

phillimorland commented on docs/currentState.md:
```diff
@@ -0,0 +1,110 @@
+Not currently working/in progress:
+- error handling for posting an idea (we do not currently show the error the API gives us?!, but handle the errors in the IdeaCreate component)
+- posting formData. We need that to post images. PR starting that: https://github.com/liqd/zt-app/pull/134
+- having the descrition in it's own component with a bigger textfield to open and close
```
however has minimum 20 char on it, needs updating?

phillimorland reviewed (commented):
super cool!

fuzzylogic2000 commented:
@phillimorland I added your remarks, also the ones for @khamui s PR. And I changed the comment stuff as that will be fixed when the PR is merged. I think we can merge this now?

khamui commented:
thanks @fuzzylogic2000 ! also thanks for reviewing the coding guide @phillimorland.
Good documentation for the next sprint app!

Rineee commented:
Also thanks from me for documenting! Super cool! :100:
